src/pages/theming/dark-mode.md

```diff
@@ -116,6 +116,23 @@ function checkToggle(shouldCheck) {
 <!-- Codepen https://codepen.io/ionic/pen/zYOpQLj -->
 <docs-codepen preview="false" user="ionic" slug="zYOpQLj" height="600px" default-tab="js,result"></docs-codepen>
 
+## Adjusting System UI Components
+
+When developing a dark theme, you may notice that certain system UI components are not adjusting to dark mode properly. To fix this you will need to specify the `color-scheme`. At the time of writing, this is supported in Safari and will be implemented in Chrome soon.
```

Comment:
Instead of saying at the time of writing, since this will be outdated in no time, can we link to a page that shows support? Maybe this one: https://www.chromestatus.com/feature/5330651267989504